guides/configuration.md

```diff
@@ -75,16 +78,6 @@ module.exports = {
 
 This section is primarily of interest to those implementing custom instrumentation or doing development on the `appoptics-apm` agent and SDK.
 
-#### Configuration File ####
-
-These configuration file properties may be set:
-
-| Property Name        | Default  | Description |
-| -------------------- | -------- | ----------- |
-|traceMode|auto|Mode 'enabled' will cause Appoptics to sample as many requests as possible. Mode 'disabled' will disable sampling and metrics.|
```

Comment:
maybe should put enabled in the default column as opposed to leaving it blank?

Reply:
That makes sense, updated.